test/framework/cloudstack.go

```diff
@@ -38,6 +38,9 @@ const (
 	cloudstackSshAuthorizedKeyVar      = "T_CLOUDSTACK_SSH_AUTHORIZED_KEY"
 	cloudstackTemplateRedhat123Var     = "T_CLOUDSTACK_TEMPLATE_REDHAT_1_23"
 	cloudstackTemplateRedhat124Var     = "T_CLOUDSTACK_TEMPLATE_REDHAT_1_24"
+	cloudstackTemplateRedhat125Var     = "T_CLOUDSTACK_TEMPLATE_REDHAT_1_25"
+	cloudstackTemplateRedhat126Var     = "T_CLOUDSTACK_TEMPLATE_REDHAT_1_26"
+	cloudstackTemplateRedhat127Var     = "T_CLOUDSTACK_TEMPLATE_REDHAT_1_27"
```

Comment:
Let's remove these and just rely on the infer logic from `templateForDevRelease` function to determine the template name